src/index.html

```diff
@@ -11,7 +11,177 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      class="form"
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+
```

Comment:
There shouldn't be empty line between parent and first child

Reply:
You should have empty lines only between siblings on the same level, not between parent and child, here's an example
```html
<section>
  <div>
    <span>First Child of Div</span>

    <span>Second Child of Div</span>
  <div>
</section>
```

Reply:
An exemption to that rule would be an ul
```html
<ul>
  <li></li>
  <li></li>
  <li></li>
  <li></li>
</ul>
```

Reply:
In the rest of your markup doesn't adhere to these rules so adjust accordingly, also take a look at the [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) and also adhere to it's rules